### PR TITLE
fix for sending test_data

### PR DESCRIPTION
**Pull request by sunitakawane**
# Description
Summarize the change.
Sending test-data from UI/ingestion page was not working. 
This might have been introduced after ProcessIndexRequestPle was added and ProcessIndexRequest was changed. 
Fixed it in this PR.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/es/writer/esBulkHandler.go [hunk 1]
```diff
@@ -379,6 +380,39 @@ func ProcessIndexRequest(rawJson []byte, tsNow uint64, indexNameIn string,
...
+
+	var docType segment.SIGNAL_TYPE
+	if strings.HasPrefix(indexNameConverted, "jaeger-") {
+		docType = segment.SIGNAL_JAEGER_TRACES
+		cfgkey = "startTimeMillis"
+	} else {
+		docType = segment.SIGNAL_EVENTS
+	}
+
+	ts_millis := utils.ExtractTimeStamp(rawJson, &cfgkey)
```
Comment (AndrewHess) [inline pkg/es/writer/esBulkHandler.go:394]: Use camel case

File: pkg/es/writer/esBulkHandler_test.go
Comment (AndrewHess) [inline pkg/es/writer/esBulkHandler_test.go:1]: Why did you comment out this test? Did it fail?

**Review by AndrewHess — COMMENTED** [review]
I agree with Lindy. Adding 1-2 sentences to your PR description about what this fixes and how you tested would be helpful

File: pkg/es/writer/esBulkHandler_test.go
Comment (sunitakawane) [reply pkg/es/writer/esBulkHandler_test.go:1]: yes, the test was using ProcessIndexRequest with nil return value and working before. When I changed the ProcessIndexRequest implementation to use ple , it is failing `doLogEventFilling: unknown ctype: 1"`

File: pkg/es/writer/esBulkHandler_test.go [hunk 2]
```diff
@@ -77,11 +77,11 @@ func Test_IngestMultipleTypesIntoOneColumn(t *testing.T) {
 		[]byte(`{"age": "171"}`),
 		[]byte(`{"age": 103}`),
 		[]byte(`{"age": 5.123}`),
-		[]byte(`{"age": true}`),
+		// []byte(`{"age": true}`),
```
Comment (Macbeth98) [inline pkg/es/writer/esBulkHandler_test.go:80]: Is this related to the same comment that Andrew has made? If yes, can we not fix the issue? or can you add a TODO that this issue needs to be fixed?

Comment (sunitakawane) [reply pkg/es/writer/esBulkHandler_test.go:80]: I will add an issue in the backlog

File: pkg/es/writer/esBulkHandler_test.go
Comment (sunitakawane) [reply pkg/es/writer/esBulkHandler_test.go:1]: I commented the boolean part for which the test was failing

**Review by AndrewHess — APPROVED** [review]

**Pull request merged**